duckduckgo/Android
# Eagerly fetch app list when the widget interstitial screen is shown

Task/Issue URL: https://app.asana.com/0/414730916066338/1168625961103601
Tech Design URL: 
CC: 

**Description**:
Eagerly fetch app list for interstitial

**Steps to test this PR**:
1. Launch interstitial from widget
1. Enter query; verify this works as intended and app list available quickly for filtering


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
